### ISSUE TITLE
Add chat bot slot validation

### ISSUE THREAD

**Issue by github1** [github1/ccnext#1]
- In the case a Lex bot, configure a pass through lambda function which delegates the validation to a 'public facing' validation api/hook on the platform service
- The underlaying validation logic and response should be agnostic to Lex (or any other chat provider)

**Comment by github1** [comment]
Is this done?

**Comment by tbouchik** [comment]
Not really. I still have to see what to do with the built-in slot types like AMAZON.NUMBER, AMAZON.FOUR_DIGIT_NUMBER... whether they are automatically validated by Lex or not.
If we can take care of the whole validation that would be better for fungibility purposes. 
And also since we haven't any "enumerationValues" property for theses types of slots, we must decide whether we should just  check if the user input corresponds to the "slotType" or whether we have to check for instance if the four digit numbers matches what we would have in a potential database, or maybe just a set of enumeration values.
